plutus-core/plutus-core/src/PlutusCore/Crypto/BLS12_381/G1.hs

```diff
@@ -170,3 +171,10 @@ memSizeBytes = BlstBindings.Internal.sizePoint (Proxy @BlstBindings.Curve1)
 -- | Compressed size of a G1 point (48 bytes)
 compressedSizeBytes :: Int
 compressedSizeBytes = BlstBindings.Internal.compressedSizePoint (Proxy @BlstBindings.Curve1)
+
+-- | Multi-scalar multiplication of G1 points.
+-- Uses MSM when the number of pairs exceeds a threshold (currently 10).
```

Comment:
@kwxm when costing, we should run the bench against different thresholds here. This 10 is an arbitrary value I randomly picked, I remember from the initial rust benchmarking @dkaidalov did, this threshold was 30, but I am unsure. Do you remember Dmytro?